frontend/src/components/app/QuickActions.js

```diff
@@ -249,7 +249,11 @@ export class QuickActions extends Component {
             const relatedActions =
               resp && resp.data ? respRel.data.actions : null;
 
-            if ((parentView.viewId || childView.viewId) && relatedActions) {
+            if (
+              (parentView.viewId || childView.viewId) &&
+              relatedActions &&
+              !_.isEmpty(parentView)
```

Comment:
Maybe we sholud just set `parentView` to null if it doesn't exist ? We won't have to make those computation-heavy checks here.

Reply:
agree

Reply:
Already explained to @siemiatj why that check is needed. Along with the <Table component refactoring which follows to be done this is most probably going to be gone.